great_expectations/data_context/types/base.py

```diff
@@ -3136,7 +3136,7 @@ def resolve_config_using_acceptable_arguments(
             run_id and run_time
         ), "Please provide either a run_id or run_name and/or run_time."
 
-        run_time = run_time or datetime.datetime.now()
+        run_time = run_time or datetime.datetime.now().astimezone()
```

Comment:
Is there specific reason to attach local timezone information here vs creating an aware datetime with utc as its timezone, e.g. `datetime.now(timezone.utc)`?

Reply:
I think it's that we *do* want things to display in local timezone where possible; the current behavior uses the local time. It's just that this is the tz-aware local timezone.

Reply:
Where are we using this information to display in local timezones? If it's in a browser, don't we want the browser's idea of local time instead of whatever the execution environment thinks local time is?

Reply:
It comes up in the MS Teams notification that is changed in this test, and in the dictionary-view of results in a notebook. I don't really have a dogmatic view on whether the internal representation is UTC or not as long as it's timezone aware.

If we were to go with `run_time = run_time or datetime.datetime.now(tz=datetime.timezone.utc)` here instead, for example, we'd get the same (correct-as-this-fix) behavior in cloud, but the dictionary view would show the correct UTC time instead of the correct local time.

Reply:
...and, what-the-heck that makes more sense to me, so I switched to that.